.github/workflows/blockchain-link-test.yml

```diff
@@ -22,11 +22,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [
-            # todo: not sure if we wan't to test other versions of node?
-            # 14.x,
-            16.x,
-          ]
+        node-version: [18.x]
```

Comment:
I am trying blockchain-link tests locally but they don't pass at the moment.

```
 tests/integration/worker/index.ts:10:20 - error TS2351: This expression is not constructable.
      Type 'typeof Worker' has no construct signatures.

    10         return new TinyWorker(() => {
                          ~~~~~~~~~~

```

So I suggest just bumping this version here so that we don't have any node 16 anywhere and fix it later